api/envoy/api/v2/auth/cert.proto

```diff
@@ -216,6 +216,10 @@ message CertificateValidationContext {
 
   // If specified, Envoy will not reject expired certificates.
   bool allow_expired_certificate = 8;
+
+  // If specified, Envoy will use this regular expression while matching :ref:`Subject Alternative
+  // Names <envoy_api_field_auth.CertificateValidationContext.verify_subject_alt_name>`.
+  string san_validation_regex = 9;
```

Comment:
In this file, please put this directly below "verify_subject_alt_name" for readability, and add a comment at the top of the struct for next field:
`// [#comment:next free field: 10]`

Reply:
I think you need to clearly define how this interacts with `verify_subject_alt_name`. In the current implementation, if the new setting is supplied, `verify_subject_alt_name` doesn't have any effect. My first thought on how they should interact is that a match on either should be considered valid. Another option is to only allow one of the two settings to be specified and reject config that has both of them. Silently ignoring one in favor of the other seems too confusing for a user.